src/components/feedback/sentry.js

```diff
@@ -0,0 +1,52 @@
+import * as Sentry from '@sentry/browser';
+import env from '../analytics-shell/env';
+
+const { feedbackLimit } = require('./components/forms.js');
+
+// Send feedback to Sentry for triaging
+export function sendToSentry({ state, props }) {
+  const { site, section, preferredLanguage, feedbackSentryDsn } = props;
+  // If feedbackSentryDsn is set to false, do not submit to Sentry
+  if (feedbackSentryDsn === false) return;
+  const { helpful, user, feedback, category, categoryType } = state;
+  const referrer = 'referrer' in document;
+  // initialize Sentry project to send the feedback
+  Sentry.init({
+    dsn: feedbackSentryDsn,
+    maxValueLength: feedbackLimit,
+    environment: env()
+  });
+  // set user attributes (if available)
+  if (user) {
+    Sentry.setUser({
+      ...(user.id && { username: user.id }),
+      ...(user.plan &&
+        user.plan.id && {
+          data: { plan: user.plan.id }
+        })
+    });
+  }
+  // configure data to send with feeedback
+  Sentry.configureScope((scope) => {
+    // set tag for site name
+    scope.setTag('site', site);
+    // set tag for category
+    scope.setTag('category', category);
```

Comment:
Not sure why the `category` and `categoryType` do not match in this [test feedback in Sentry](https://sentry.io/organizations/mapboxcom/issues/2498147667/?environment=staging&project=1848287&query=is%3Aunresolved). `category` is `Something is confusing` while `categoryType` is all the `I like this..` options. 

![image](https://user-images.githubusercontent.com/10479155/124818509-bbf8c880-df30-11eb-8e8a-7016687f3ba9.png)

I was clicking a lot of things when I was testing, but there are only three events with this `sessionId` and the only `category` that appears is `Something is confusing`.

Reply:
I'll look into this!

Reply:
I wonder if you tested "I like this" with the first /Feedback test case and then (without refreshing) tested the other test case with "Something is confusing" - I was able to replicate doing that. I think we can close / clear Sentry after submission. I'll see if that'll work. This will be important to get right for pages with sectioned feedback.

Reply:
OH! https://docs.sentry.io/platforms/javascript/enriching-events/scopes/#local-scopes gotta use local scopes!!!

Reply:
This should be fixed now! If you agree: smash that resolve conversation button.

Reply:
SMASH.